ts/screens/wallet/payment/ManualDataInsertionScreen.tsx

```diff
@@ -65,7 +65,8 @@ type State = Readonly<{
   organizationFiscalCode: O.Option<
     ReturnType<typeof OrganizationFiscalCode.decode>
   >;
-  inputValue: string;
+  pnnInputValue: string;
+  ofcInputValue: string;
```

Comment:
```suggestion
  noticeNumberInputValue: string;
  orgFiscalCodeInputValue: string;
```
I believe some verbosity would go a long way in improving first impact readability , do you agree?